src/zenml/enums.py

```diff
@@ -151,3 +151,15 @@ class PermissionType(StrEnum):
     WRITE = "write"  # allows the user to create, update, delete everything
     READ = "read"  # allows the user to read everything
     ME = "me"  # allows the user to self administrate (change name, password...)
+
+
+class FlavorFieldDataType(StrEnum):
```

Comment:
Dont we want a more generic name as @fa9r might reuse this for the metadata story?

Reply:
I would argue, that the Metadata story will use very different types.